cms/templates/emails/user_task_complete_email.txt

```diff
@@ -0,0 +1,13 @@
+<%! from django.utils.translation import ugettext as _ %>
+
+% if detail_url:
+
+${_("Your task {task_name} has completed with status {task_status}. To view details on your task or download any files created please copy and paste this address into your web browser's address bar:").format(task_name=task_name, task_status=task_status)}
+${detail_url}
+
+% else:
+
+${_("Your task {task_name} has completed with status {task_status}. Sign in to view the details of your task or download any files created.").format(task_name=task_name, task_status=task_status)}
```

Comment:
Should probably run the wording of this whole message by somebody in Product.  Marco seemed pretty reluctant to use generic-to-any-task phrasing like this in the UI, they may prefer something more tailored like "Your import of course {name} succeeded, click here to view it in {settings.STUDIO_NAME}: {url}"